style.css

```diff
@@ -0,0 +1,1032 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&family=Quicksand:wght@300;400;500;600;700&display=swap');
```

Comment:
- [ ] I noticed that you used only Lato as a font. Please note that this project uses both Fonts used originally, Cocogoose, and Lato 👍